Products/CMFPlone/browser/icons.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+from lxml import etree
```

Comment:
@petschki It seems CMFPlone has lxml as an undeclared dependency. This is one of the imports that currently causes trouble. Though, this undeclared dependency has been added some time ago here:

https://github.com/plone/Products.CMFPlone/commit/946f3b3cee049646d3cf1eb0f176e9cb3c743a54#r60340299